src/app/components/utility/side-bar/side-bar.component.scss

```diff
@@ -7,6 +7,7 @@
   color: #fff;
   z-index: 10;
   text-align: center;
+  padding-bottom: 34px;
```

Comment:
Instead I would suggest to revert to [this](https://github.com/Cloud-CV/EvalAI-ngx/pull/277/commits/b73cafad8c50ed0cb0c566424d73b94ec6fb6141) commit.